prisma/schema.prisma

```diff
@@ -35,16 +35,17 @@ enum Category {
 }
 
 model Proposal {
-    id            String       @id @default(cuid())
-    name          String       @db.VarChar(200)
-    comment       String       @db.VarChar(2000)
-    accepted      Boolean      @default(false)
-    createdAt     DateTime     @default(now())
-    occupation    String       @db.VarChar(200)
-    twitterURI    String
-    websiteURI    String
-    Partnership   Partnership? @relation(fields: [partnershipId], references: [id])
-    partnershipId String?
+    id             String       @id @default(cuid())
+    name           String       @db.VarChar(200)
+    comment        String       @db.VarChar(2000)
+    createdAt      DateTime     @default(now())
+    occupation     String       @db.VarChar(200)
+    twitterURI     String
+    websiteURI     String
+    signature      String?
+    partnerAddress String
```

Comment:
What is the point? Partnership has ownerAddress field

Reply:
if `account.address === partnership.ownerAddress` we should show sign button
if `account.address === proposal. partnerAddress  && proposal.signature` we should show mint button for partner